src/ontology/uberon-edit.obo

```diff
@@ -214663,10 +214663,13 @@ property_value: provenance_notes "This class was sourced from an external ontolo
 
 [Term]
 id: UBERON:3011045
-name: gasserian ganglion
+name: obsolete gasserian ganglion
 synonym: "gasserian ganglia" RELATED [UBERON:3011045]
 is_a: UBERON:0000045 ! ganglion
```

Comment:
There cannot be is_a (or subClassOf), see https://oboacademy.github.io/obook/howto/obsolete-term/

Reply:
Aah item 8 missed that. Will do thanks for catching this

Reply:
Looks good :)